sys/sys/fcntl.h

```diff
@@ -143,6 +143,10 @@ typedef	__pid_t		pid_t;
 #define	O_NAMEDATTR	0x04000000	/* NFSv4 named attributes */
 #endif
 
+#if __POSIX_VISIBLE >= 202405 || __BSD_VISIBLE
```

Comment:
No need to specify __BSD_VISIBLE.  If _POSIX_C_SOURCE is not enforced, we already set __POSIX_VISIBLE to max supported version.